.github/workflows/redeploy/config.yaml

```diff
@@ -0,0 +1,192 @@
+# Application configuration file
```

Comment:
Don't forget to remove this file if you decide to use environment variables instead.

Reply:
oh, I think I will use this approach. Updating env for each machine is more tedious to me and put everything in config.yml makes it more intuitive to other users.

Reply:
@giongto35, I meant that it's better to override a couple of config params by defining and passing environment vars into the Bash script and then inside Docker. The reason behind that: we're removing config file copying or modifying work which we have to do now every time we change default config.yaml in the app, that file plays the role of the app config structure initializatior. Internal config structure is initialized with empty values (0, '', nil) without it, which will cause a crash. So, we better leave the default config.yaml inside the Docker image and just override needed values externally with env vars. We can do that in your Bash script in the repo here.
I'll try to fix this later.

Reply:
Yes, I expect that the default config.yaml will be changed quite often.
You use Docker image or build with the default config inside and just override needed params with env variables, flags, whatever. That's how I see it. In your example just set system env CLOUD_GAME_ENVIROMENT=prod on your server.
And yes, it is better to keep all the config params together not scattered all around the whole codebase.
How do you see it?